cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, pr_remote, commit_sha1, branches,
         if dry_run:
             click.echo("Dry run requested, listing expected command sequence")
 
+        self.remember_current_branch()
```

Comment:
The method is called only once, let's inline it.

Reply:
I think named things are better readable within the code flow, don't you?

Reply:
Well, I don't care too much.
My the only objection is the call *hides* `_start_branch` initialization by moving attribute assignment from constructor to method.
Also, cherry-picker doesn't use underscores for attribute names.
Let's rename `self._start_branch` for consistency.

Reply:
Yeah, I've also had this concern and then saw this pattern being used all over the place. I think, I can simplify this.